resources/augeas.go

```diff
@@ -230,6 +236,7 @@ func (obj *AugeasRes) CheckApply(apply bool) (checkOK bool, err error) {
 		return false, errwrap.Wrapf(err, "augeas: error while saving augeas values")
 	}
 
+	// XXX: recurse? not a good idea. Instead test for file existence manually.
```

Comment:
recurse with apply = False. So it will never be called by the function that uses it.

This will not only catch if the file does not exist, but it will also catch other problems. e.g.:

```
set:
  - ['ForwardX11', 'no']
  - ['ForwardX11', 'yes']
```

That will be detected by this recurse call as an error. it is a great way to detect that the resource has been applied successfully. I am not sure I want to remove that even if the upstream bug was fixed.